spec/controllers/recordings_controller_spec.rb

```diff
@@ -27,6 +27,15 @@
       response_recording_ids = JSON.parse(response.body)['data'].map { |recording| recording['id'] }
       expect(response_recording_ids).to be_empty
     end
+
+    it 'returns the proper recordings according to the query' do
+      recordings = create_list(:recording, 6)
+      create_list(:room, 5, user:, recordings:)
+
+      get :index, params: { q: recordings[0].name }
+      response_recording_id = JSON.parse(response.body)['data'][0]['id']
+      expect(response_recording_id).to eq(recordings[0].id)
```

Comment:
I think it would be better if this recording was a bit more explicit. Create 5 recordings and manually structure the names so that some have text in common